### ISSUE TITLE
How do I get ratings from the business response object?

### ISSUE THREAD

**Issue by abhidotnet** [Yelp/yelp-api#92]
This is my code. 

```
params = {
    'lang': 'en'
}
response2 = c.get_business('alice-bakery-and-confectionary-north-wales-2', **params)
print response2.business.rating_img_url, response2.business.location.address, response2.business.name, response2.business.display_phone, response2.business.review_count
```
I am getting the following error.
```AttributeError: 'Business' object has no attribute 'rating_img_url'```

I am referring to the following documentation.
https://www.yelp.com/developers/documentation/v2/business

**Comment by mittonk** [comment]
@abhidotnet : Are you using the Python clientlib over at https://github.com/Yelp/yelp-python ? 

This sounds a lot like you're hitting https://github.com/Yelp/yelp-python/issues/27, but let's move the discussion over there if you're using the clientlib.

**Comment by abhidotnet** [comment]
@mittonk Yes. I am using the Python client lib at https://github.com/Yelp/yelp-python

**Comment by abhidotnet** [comment]
I think this is the way to get it.

```response2.business.rating.rating```

**Comment by watterso** [comment]
@abhidotnet just a heads up, I released version 1.0.1 which made some changes to the clientlib. If you upgrade, getting the rating is a little simpler:
```python
>>> alice_bakery = client.get_business('alice-bakery-and-confectionary-north-wales-2', lang='en')
>>> alice_bakery.business.rating
4.5
>>> alice_bakery.business.rating_img_url
u'http://s3-media2.fl.yelpcdn.com/assets/2/www/img/99493c12711e/ico/stars/v1/stars_4_half.png'
```
